Packs/CommonPlaybooks/Playbooks/playbook-Search_For_Hash_In_Sandbox_-_Generic.yml

```diff
@@ -576,15 +424,16 @@ tasks:
       format:
         simple: pdf
       sha256:
-        simple: ${inputs.FileSha256}
+        complex:
+          root: inputs.FileSha256
```

Comment:
That's actually interesting why we are not using the `WildFire.Report.SHA25`
In case there is a good reason, let's just add a unique transformer there.

Reply:
No changes were made to task (23) inputs since it checks for WildFire reports and retrieves them in PDF format. 
The only change I made to task number 23 is adding the 'unique' transformer to the currently used input.